src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1902,7 +1902,7 @@ private BoundBaseReference BindBase(BaseExpressionSyntax node, DiagnosticBag dia
                 Error(diagnostics, ErrorCode.ERR_NoBaseClass, node);
                 hasErrors = true;
             }
-            else if (this.ContainingType is null || (node.Parent.Kind() != SyntaxKind.SimpleMemberAccessExpression && node.Parent.Kind() != SyntaxKind.ElementAccessExpression))
+            else if (this.ContainingType is null || (node.Parent != null && node.Parent.Kind() != SyntaxKind.SimpleMemberAccessExpression && node.Parent.Kind() != SyntaxKind.ElementAccessExpression))
```

Comment:
>node.Parent != null [](start = 53, length = 19)

I think we should get into this ```if``` when parent is null, standalone base expression is not legal